tpot/tpot.py

```diff
@@ -235,6 +250,17 @@ def fit(self, features, classes):
         None
 
         """
+        if self.expert_source != None:
+            input_ekf_source = pd.read_csv(self.expert_source, sep='\t')
```

Comment:
Expert knowledge sources won't be passed as files. They will be passed as lists of numpy arrays.